src/pages/Login.tsx

```diff
@@ -0,0 +1,54 @@
+import { Card, CardContent, Grid, Typography } from '@mui/material';
```

Comment:
The code in this file was copied and pasted from _src/app/Unauthenticated.tsx_. Please let me know if you see any inconsistencies.